web-app/HeimScripts/heatmap/run.R

```diff
@@ -95,6 +154,40 @@ addStats <- function(df, sorting) {
   return(df)
 }
 
+# Coefficient of variation
+coVar     <- function(x) ( sd(x)/mean(x) )
```

Comment:
This name might be confusing: there is also something called "covariance", which is something else. Rename?